scripts/package-build/blackbox_exporter/build.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+CWD=$(pwd)
+set -e
+
+BUILD_ARCH=$(dpkg-architecture -qDEB_TARGET_ARCH)
+
+SRC="blackbox_exporter"
+if [ ! -d ${SRC} ]; then
+    echo "Source directory does not exist, please 'git clone'"
+    exit 1
+fi
+
+cd $SRC
+
+mkdir -p debian
+
+echo "I: Create $SRC/debian/control"
+cat <<EOF > debian/control
+Source: blackbox-exporter
+Section: net
+Priority: optional
+Maintainer: VyOS Package Maintainers <maintainers@vyos.net>
+Build-Depends: debhelper-compat (= 13)
+Standards-Version: 4.5.1
+Homepage: https://github.com/prometheus/blackbox_exporter
+
+Package: blackbox-exporter
+Architecture: ${BUILD_ARCH}
+Depends: \${shlibs:Depends}, \${misc:Depends}
+Description: The blackbox exporter allows blackbox probing of endpoints over HTTP, HTTPS, DNS, TCP, ICMP and gRPC.
+EOF
+
+echo "I: Create $SRC/debian/changelog"
```

Comment:
Can we avoid embedding data in shell scripts to make it easier to edit and keep track of? I think we might include the packaging as a patch so that it can be applied by the usual patching mechanism — what do you think of it?

Reply:
I just looked at how other packages are built and tried to stich together something working. I don't really have an opinion on how this should be done, from my naive pov the fpm way looked the most straight forward.

But having build related files as patches seems counter intuitive, I wouldn't have looked there.

Reply:
Lets use this as a starting point for the upcoming FPM -> Debian builder replacement